<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Принятые пачки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <column>
       <property name="text">
        <string>Крой</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Пачка</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Артикул</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Клиент</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата прием</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Дата С</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_date_from">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>По</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_date_to">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_calc()</slot>
 </slots>
</ui>
